language/resource.language.en_gb/strings.po

```diff
@@ -216,6 +216,10 @@ msgctxt "#748"
 msgid "Change the PIN Lock for @DISTRONAME@ Settings access"
 msgstr ""
 
+msgctxt "#749"
+msgid "Custom set wireless hardware frequencies to comply with regional laws."
```

Comment:
Overrride the Wireless Regulatory Domain so radio properties (channels and transmit power) comply with local laws or match the country configuration of your router.